src/http/request.cr

```diff
@@ -80,6 +81,24 @@ class HTTP::Request
     @query_params ||= uri.query_params
   end
 
+  # Returns a convenience wrapper to parse form params, see `URI::Params`.
+  # Returns `nil` in case the content type `"application/x-www-form-urlencoded"`
+  # is not present or the body is `nil`.
+  def form_params? : HTTP::Params?
+    @form_params ||= begin
+      if headers["Content-Type"]? == "application/x-www-form-urlencoded"
+        if _body = body
+          HTTP::Params.parse(_body.gets_to_end)
+        end
```

Comment:
Variables starting with an underscore are often used for unused variables, and some linters will pick this up. Simply use a method call on `self` here:
```suggestion
        if body = self.body
          HTTP::Params.parse(body.gets_to_end)
        end
```

Reply:
Thanks for your review.